analysis/src/Hover.ml

```diff
@@ -100,12 +100,12 @@ let hoverWithExpandedTypes ~docstring ~file ~package ~supportsMarkdownLinks typ
                Markdown.goToDefinitionText ~env ~pos:loc.Warnings.loc_start
              else ""
            in
-           "\n" ^ Markdown.spacing
+           Markdown.divider ^ (if supportsMarkdownLinks then Markdown.spacing else "")
```

Comment:
We should probably name `supportsMarkdownLinks` something smarter now that it handles more cases, but that's a question for later.